spot-client/src/spot-tv/ui/components/meeting-frame/JitsiMeetingFrame/JitsiMeetingFrame.js

```diff
@@ -328,6 +333,31 @@ export class JitsiMeetingFrame extends AbstractMeetingFrame {
         }
     }
 
+    /**
+     * Callback invoked when a participant has raised their hand.
+     *
+     * @param {Object} event - The event returned from the external api.
+     * @param {string} event.id - The endpoint ID of the participant.
+     * @param {number} event.handRaised - Timestamp of when the hand was raised or 0 if not raised.
+     * @private
+     * @returns {void}
+     */
+    _onRaiseHandChange({ id, handRaised }) {
+        // Ignoring raise hand change for non local participant;
+        if (this._localParticipantId !== id) {
```

Comment:
`id` could be falsey if sent before the meeting join completes IIRC, since we don't know the local participant ID in advance.